weblate/utils/templatetags/icons.py

```diff
@@ -38,10 +37,21 @@
 
 @register.simple_tag()
 def icon(name, fallback=None):
-    name = name or fallback
+    if fallback is not None:
+        name = fallback
+
+    if not name:
+        return ""
+
     if name not in CACHE:
-        with open(os.path.join(PATH, name), "r") as handle:
-            CACHE[name] = mark_safe(handle.read())
+        icon_file = os.path.join(settings.STATIC_ROOT, "icons", name)
+        try:
+            with open(icon_file, "r") as handle:
+                CACHE[name] = mark_safe(handle.read())
+        except OSError as e:
+            LOGGER.error(str(e))
```

Comment:
Using `report_error` would be more appropriate here, that way the error would correctly show up in Sentry/Rollbar if these are configured. It takes care of the logging as well.

```suggestion
            report_error(error, prefix="Failed to load icon")
```